index.html

```diff
@@ -45,6 +45,170 @@ <h2 class="secondary-titel">
           <a href="https://www.github.com/youmari" class="github-link"><img src="./images/github.svg" alt="github icon"></a>

```

Comment:
[ OPTIONAL ] Please make sure all linter checks are passing before asking for a review.